tsconfig.json

```diff
@@ -9,6 +9,7 @@
 		"importHelpers": true,
 		"incremental": true,
 		"inlineSources": true,
+		// "isolatedDeclarations": true,
```

Comment:
We will enable `isolatedDeclarations` as soon as https://github.com/microsoft/rushstack/issues/4805 is resolved.

Reply:
Comments are valid in TS config files, but I've a `TODO` with the a link to the external issue.